### ISSUE TITLE
Status topics are not published

### ISSUE THREAD

**Issue by LordVoodoo** [svrooij/sonos2mqtt#148]
Having a problem I cannot handle:

I have setup sonos2mqtt inside a Docker-Container including a MacVLAN into the host network. The model is noticed by the MQTT broker (Mosquitto) and delivers "homeassistant" as topic.
For some reason, the speakers, mainly Play:1, do not deliver any status / title / track information.

At a second place, the setup works fine. This time only having Sonos Beams.

Any idea how to handle this?

**Comment by svrooij** [comment]
We need some more info. Can you add your environment variables?
Have you tried listening for `sonos/#`

**Comment by LordVoodoo** [comment]
Hello,

for the configuration:
- several Play:1 devices, all wireless
- one Playbar, wired

No 5Ghz device in the setup. I installed sonos2mqtt in Docker with net-host-option.

For Monitoring I used „MQTT Explorer“ on Mac to see all messages on the Broker.

I received all homeassistant entries and I could publish messages to speakers, tried it with a URI from a working setup at a different place.
Although, the state is not reported.

Viele Grüße, Matthias
